rekcurd.proto

```diff
@@ -78,7 +75,7 @@ message DefaultData {
         google.protobuf.ListValue ndarray = 3;
         bytes binData = 4;
         string strData = 5;
-        StringList strDataList = 6;
+        google.protobuf.Value jsonData = 6;
```

Comment:
According to [the official doc](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/Value), `Value` represents a dynamically typed value which can be either null, a number, a string, a boolean, a recursive struct value, or a list of values.